db/migrate/20200327142848_create_caution_flag_rules.rb

```diff
@@ -0,0 +1,20 @@
+class CreateCautionFlagRules < ActiveRecord::Migration[5.2]
+  def up
+    create_table :caution_flag_rules do |t|
+      t.integer :rule_id
```

Comment:
Rails makes these foreign keys `bigint`s now.  And this doesn't get you an index on `rule_id`.  Try this instead:
```ruby
    create_table :caution_flag_rules do |t|
      t.belongs_to :rule, foreign_key: true
      t.string :title
      t.string :description
      t.string :link_text
      t.string :link_url
      t.timestamps
    end
```
and you don't need the `add_foreign_key` below either.

Reply:
changed